x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/new_index/method_connector/method_connector.tsx

```diff
@@ -132,13 +136,20 @@ export const MethodConnector: React.FC = () => {
             children: (
               <EuiText size="s">
                 <p>
-                  {i18n.translate(
-                    'xpack.enterpriseSearch.content.newIndex.steps.createIndex.content',
-                    {
-                      defaultMessage:
-                        'Provide a unique name for your index and select an optional index language.',
-                    }
-                  )}
+                  <FormattedMessage
+                    id="xpack.enterpriseSearch.content.newIndex.methodConnector.steps.createConnectorIndex.content"
+                    defaultMessage="Provide a unique index name and optionally set a default {link} for the index. This index will hold your data source content, and is optimized with default field mappings for relevant search experiences."
```

Comment:
I try to use more descriptive names for variables in i18n strings, to help out our translators.  Maybe something like "languageAnalyzerDocLink"